src/tree_schema.c

```diff
@@ -215,12 +215,14 @@ lys_getnext_into_case(const struct lysc_node_case *first_case, const struct lysc
 }
 
 API const struct lysc_node *
-lys_getnext(const struct lysc_node *last, const struct lysc_node *parent, const struct lysc_module *module, uint32_t options)
+lys_getnext_(const struct lysc_node *last, const struct lysc_node *parent, const struct lysc_module *module,
```

Comment:
I am guessing this should be a static function now.

Reply:
yes, and description is missing